x-pack/plugins/ml/public/application/services/usage_collection.test.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UsageCollectionSetup } from 'src/plugins/usage_collection/public';
+
+import { mlUsageCollectionProvider } from './usage_collection';
+
+describe('usage_collection', () => {
+  let usageCollection: UsageCollectionSetup | null = null;
+
+  beforeEach(() => {
+    usageCollection = ({
+      reportUiCounter: jest.fn(),
+    } as unknown) as UsageCollectionSetup;
```

Comment:
use `jest.Mocked` and you can remove all ts-ignore comments
```suggestion
    } as unknown) as jest.Mocked<UsageCollectionSetup>;
```

Reply:
Changed in https://github.com/elastic/kibana/pull/107772/commits/b622cfd2a80be91a587e2db557bdc983c6f934a6